### PR TITLE
Branding and theme change

### PR DESCRIPTION
**Pull request by ariansobczak-rst**
Introduced Theme generator, dark and light theme for AdminJS.
Related with https://github.com/SoftwareBrothers/adminjs-design-system/pull/27
Related with https://github.com/SoftwareBrothers/adminjs-themes/pull/1

**Dark theme**
image
**Light theme**
image

### CODE REVIEW

**Comment by ariansobczak-rst** [comment]
Build failed because of lower `@adminjs/design-theme`, will be updated after PR https://github.com/SoftwareBrothers/adminjs-design-system/pull/27

File: src/frontend/components/app/branding-selector.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+    return acc;
+  }, [] as SelectableBranding[]);
+
+  if (!selectableBrandings.length) return null;
+
+  return (
+    <SelectContainer padding="xl">
+      <Select
+        placeholder="Select theme..."
```
Comment (dziraf) [inline src/frontend/components/app/branding-selector.tsx:66]: This should go through i18n imo

File: src/frontend/store/actions/initialize-branding.ts [hunk 2]
```diff
@@ -1,13 +1,37 @@
-import { BrandingOptions } from '../../../adminjs-options.interface'
+import { AdminJSOptions, BrandingOptions } from '../../../adminjs-options.interface';
```
Comment (dziraf) [inline src/frontend/store/actions/initialize-branding.ts:1]: doesn't core repo's eslint enforce "no semicolon"?

File: src/backend/decorators/branding/branding.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { BrandingOptions, BrandingOptionsFunction } from 'src'
+import AdminJS from '../../../adminjs'
+import { CurrentAdmin } from '../../../current-admin.interface'
+
+class Branding {
```
Comment (dziraf) [inline src/backend/decorators/branding/branding.ts:5]: this is currently not used afaik?

File: src/frontend/store/actions/initialize-branding.ts (hunk 2 above)
Comment (ariansobczak-rst) [reply src/frontend/store/actions/initialize-branding.ts:1]: Good point, introduced prettier `"semi": false` and will be added in eslint in dev environment

File: src/frontend/components/app/branding-selector.tsx (hunk 1 above)
Comment (ariansobczak-rst) [reply src/frontend/components/app/branding-selector.tsx]: Moved as message `brandingSelectorSearch`

**Comment by VasilNikolov** [comment]
Hi, Is this functionality going to be merged?

**Comment by dziraf** [comment]
Closing in favour of AdminJS v7 which will have a separate frontend.

**Pull request closed without merging**